.github/workflows/server-workflow.yml

```
@@ -25,15 +25,15 @@ jobs:
-  #     image: epitechcontent/epitest-docker
-  #   steps:
-  #     - name: Checkout code
-  #       uses: actions/checkout@v4
-  #
-  #     - name: Run tests
-  #       run: cd server && make tests
+  check_tests:
+    needs: [check_compilation]
+    name: Run server tests
+    runs-on: ubuntu-latest
+    container: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)